common/pom.xml

```diff
@@ -2,65 +2,74 @@
-        <version>1.12</version>
-        <executions>
-          <execution>
-            <id>attach-artifact</id>
-            <phase>package</phase>
-            <goals>
-              <goal>attach-artifact</goal>
-            </goals>
-            <configuration>
-              <artifacts>
-                <artifact>
-                  <file>${project.basedir}/src/main/resources/docker/docker.bom</file>
-                  <type>bom</type>
-                  <classifier>docker-catalog</classifier>
-                </artifact>
-                <artifact>
-                  <file>${project.basedir}/src/main/resources/common/ca.bom</file>
-                  <type>bom</type>
-                  <classifier>ca-catalog</classifier>
-                </artifact>
-                <artifact>
-                  <file>${project.basedir}/src/main/resources/common/common.bom</file>
-                  <type>bom</type>
-                  <classifier>common-catalog</classifier>
-                </artifact>
-              </artifacts>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
-  </build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>1.12</version>
+                <executions>
+                    <execution>
+                        <id>attach-artifact</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>attach-artifact</goal>
+                        </goals>
+                        <configuration>
+                            <artifacts>
+                                <artifact>
+                                    <file>${project.basedir}/src/main/resources/docker/catalog.bom</file>
+                                    <type>bom</type>
+                                    <classifier>catalog</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>${project.basedir}/src/main/resources/docker/docker.bom</file>
+                                    <type>bom</type>
+                                    <classifier>docker</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>${project.basedir}/src/main/resources/common/ca.bom</file>
+                                    <type>bom</type>
+                                    <classifier>ca</classifier>
+                                </artifact>
+                                <artifact>
+                                    <file>${project.basedir}/src/main/resources/common/common.bom</file>
+                                    <type>bom</type>
+                                    <classifier>common</classifier>
```

Comment:
Agree `catalog-catalog` is ugly but i wouldnt include it as an artifact like this as catalog.bom is a noting file that us only really useful when you have the jar already 

I think it should be omitted completly WYT?